kubernetes/schema_container.go

```diff
@@ -650,22 +650,22 @@ func probeSchema() *schema.Resource {
 		Type:         schema.TypeInt,
 		Optional:     true,
 		Default:      10,
-		ValidateFunc: validatePositiveInteger,
+		ValidateFunc: validation.IntAtLeast(1),
```

Comment:
Thanks for catching this! I had lost track of which one was `> 0` vs `>= 0`. I'll get it sorted out.

Reply:
This looks correct, actually because validatePositiveInteger just checked to see that the number was at least 1.

Here's the function from the git log:
```
-func validatePositiveInteger(value interface{}, key string) (ws []string, es []error) {
-       v := value.(int)
-       if v <= 0 {
-               es = append(es, fmt.Errorf("%s must be greater than 0", key))
-       }
-       return
-}
```

Reply:
I also double-checked the API docs for this particular field. It says:

```
periodSeconds
integer	How often (in seconds) to perform the probe. Default to 10 seconds. Minimum value is 1.
```

Reply:
I see why you thought it was InitialDelaySeconds though. The diff made it look like I had modified that field, but I didn't. There's no validation for InitialDelaySeconds, since that one is just a plain int.